app.js

```diff
@@ -27,11 +18,22 @@ window.AppData = new Proxy(defaultData, {
         window.console.error('Unable to parse l10n strings: ', ex);
       }
     } else obj[prop] = value;
+
+    if (prop === 'src') {
```

Comment:
Maybe worth checking if the old 'src' matches the new 'src', and only emitting if it's actually a change